### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by joewalker**
Include Brian's post in main readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,16 +8,21 @@ Build the world’s best web browser
...
 * [XUL problems](text/0003-problems-with-xul.md) aims to list the different kinds of problems that exist with XUL.
 * [XBL and Web Components](text/0004-xbl-web-components.md) compares some old Mozilla technology (XBL) with modern Web Components.
 
 ## Posts
 
-* [Browser Architecture Update](posts/2017-07-27-browser-architecture-update.md) is the first of our newsletters, sent to [firefox-dev](https://www.mozilla.org/en-US/about/forums/#firefox-dev) to keep everyone in the loop.
+We typically send our newsletters to [firefox-dev](https://www.mozilla.org/en-US/about/forums/#firefox-dev).
+
+* [Browser Architecture Update](posts/2017-07-27-browser-architecture-update.md) is the first of our newsletters introducing our work. [[mailing-list-post](https://groups.google.com/forum/#!msg/firefox-dev/ueRILL2ppac/RxR9lLPkAwAJ)]
+* [Browser Architecture Newsletter #2](posts/2017-07-27-browser-architecture-update.md) is the follow-up with updates on XBL Conversion, Storage and Sync and Workflow Improvments [[mailing-list-post](https://groups.google.com/forum/#!topic/firefox-dev/Rc2w2a9e8HQ)]
```
Comment (bgrins) [inline README.md]: Nit: 'the follow up' -> 'a follow up'

**Review by bgrins — APPROVED** [review]
Wow that is a lot of git ignoring

**Comment by joewalker** [comment]
bgrins:
> Wow that is a lot of git ignoring

Yeah - not sure if this is an ideal way to handle gitignores, but it's something github promote, so I thought it was worth a go.

File: README.md (hunk 1 above)
Comment (joewalker) [reply README.md]: Fixed in 04aa894

**Pull request merged**